public/scripts/open-project.js

```diff
@@ -0,0 +1,115 @@
+(function() {
+  require(["jquery"], function($) {
+    var projects = document.querySelectorAll("tr.bramble-user-project");
+
+    function generateUrl(path, date) {
+      var url = window.location;
+      var qs = url.search;
+      if(qs.length < 1) {
+        qs = "?";
+      }
```

Comment:
if `qs.length > 1`, don't you need to add a `&` in order to concat the stuff on the next line to the existing bits?